chord_metadata_service/chord/views_search.py

```diff
@@ -134,6 +134,24 @@ def get_biosamples_with_experiment_details(subject_ids):
     return biosamples_exp_tissue_details
 
 
+def append_biosamples_details(biosamples_experiments_details):
```

Comment:
I'd suggest a different name for this, since it isn't appending something but rather building a new structure. also, please give this a return type.

Reply:
finally, I think these should be in some type of utility file if theyre being used elsewhere, and properly declared in `__all__`.